pkg/network/protocols/http/protocol.go

```diff
@@ -220,10 +220,15 @@ func (p *protocol) setupMapCleaner(mgr *manager.Manager) {
 func (p *protocol) GetStats() (*protocols.ProtocolStats, func()) {
 	p.eventsConsumer.Sync()
 	p.telemetry.Log()
+	stats := p.statkeeper.GetAndResetAllStats()
```

Comment:
Shouldn't id be changed also for the HTTP/2 protocol GetStats()? (as the change is being done for both http and http2)

Reply:
good catch, modified